python/fledge/services/core/api/update.py

```diff
@@ -126,10 +125,10 @@ async def get_updates(request: web.Request) -> web.Response:
         Example
          curl -sX GET http://localhost:8081/fledge/update |jq
     """
-    _platform = platform.platform()
-    update_cmd = "sudo apt update"
-    upgradable_pkgs_check_cmd = "apt list --upgradable | grep \^fledge"
-    if "centos" in _platform or "redhat" in _platform:
+    if utils.is_debian():
+        update_cmd = "sudo apt update"
+        upgradable_pkgs_check_cmd = "apt list --upgradable | grep \^fledge"
+    else:
```

Comment:
that way we can also keep here default as is; and  only `"centos" in _platform or "redhat" in _platform` with if is replaced.